packages/web/components/Technology/Details/Tabs/GeoLocation.js

```diff
@@ -36,22 +37,26 @@ const Geolocation = () => {
 	};
 
 	const getMarkers = () => {
-		return technology.terms
-			.filter(
-				({ term }) =>
-					[
-						TechonologyEnums.WHO_DEVELOP,
-						TechonologyEnums.WHERE_IS_ALREADY_IMPLEMENTED,
-					].includes(term) && markerFilters.includes(term),
-			)
-			.map(parseTermMetaIntoMarker);
+		const terms = technology?.terms || rawTerms;
+		if (terms) {
+			return terms
+				.filter(
+					({ term }) =>
+						[
+							TechonologyEnums.WHO_DEVELOP,
+							TechonologyEnums.WHERE_IS_ALREADY_IMPLEMENTED,
+						].includes(term) && markerFilters.includes(term),
+				)
+				.map(parseTermMetaIntoMarker);
+		}
+		return null;
```

Comment:
```suggestion
		const terms = technology?.terms || rawTerms;
		if (!terms) {
		     return;
		}
		
		return terms
				.filter(
					({ term }) =>
						[
							TechonologyEnums.WHO_DEVELOP,
							TechonologyEnums.WHERE_IS_ALREADY_IMPLEMENTED,
						].includes(term) && markerFilters.includes(term),
				)
				.map(parseTermMetaIntoMarker);
```


Só para melhorar um pouco a leitura, return early 🔝

Reply:
@marcusmota aqui retornei um `null` para evitar o erro do ESLint `Arrow function expected no return value. (consistent-return)`:

![image](https://user-images.githubusercontent.com/10481028/96374399-e8717800-1148-11eb-896f-05a311ce9893.png)